draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -796,14 +1087,22 @@ Constants: A and B, the parameters of the Weierstrass curve.
 
 Output: (x, y), a point on E.
 
+Sign of y: this encoding does not compute a square root, so there
+is no ambiguity regarding the sign of y.
+
+Exceptions: The only exceptional case is hash2base(alpha) == 0.
```

Comment:
Ah, I see your point.

The text currently in `{{map-exceptions}}` says "we discuss the exceptional cases and show how to handle them in constant time."

If the preference is to get rid of the explicit discussion and just tweak the maps, that's fine. But it might be nice to make explicit the tweaks and the reasons for them, otherwise readers might wonder why the descriptions in this document differ from the published algorithms...

Reply:
Ideally, the tweaks and the related discussion can be summarized in a supportive short paper, which can be released on eprint. I am happy to collaborate with that.

Reply:
(see comment above. I can see this working, but it's not clear to me why that's preferable to having the document be self-contained)